src/Symfony/Component/Notifier/Bridge/LightSms/LightSmsTransport.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LightSms;
+
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Vasilij Duško <vasilij@prado.lt>
+ */
+final class LightSmsTransport extends AbstractTransport
+{
+    protected const HOST = 'www.lightsms.com';
+
+    private $login;
+    private $password;
+    private $phone;
+
+    private const ERROR_CODES = [
+        '000' => 'Service unavailable',
+        '1' => 'Signature not specified',
```

Comment:
I would replace all "_Something_ not specified" with "Missing _Something_"
So here it would be 

```suggestion
        '1' => 'Missing Signature',
```